index.js

```diff
@@ -17,18 +24,14 @@ try {
   process.exit(1);
 }
 
-// Make sure we have a valid env
-if (process.env.AWS_SECRET_KEY_ID === undefined ||
-  process.env.AWS_SECRET_ACCESS_KEY === undefined ||
-  process.env.S3_BUCKET_NAME === undefined) {
-  console.error('Please provide valid environment variables.');
-  process.exit(1);
-}
-
 var meta_folder = process.env.META_FOLDER || 'meta';
 
 var limitParallel = 20;
 
+// Upping concurrency limit on Node <= 10
+// https://github.com/openimagerynetwork/oin-meta-generator/issues/14
+http.globalAgent.maxSockets = https.globalAgent.maxSockets = 20;
```

Comment:
Maybe do this to avoid artificially limiting it on newer versions?

```javascript
http.globalAgent.maxSockets = Math.max(http.globalAgent.maxSockets, 20);
https.globalAgent.maxSockets = Math.max(https.globalAgent.maxSockets, 20);
```